interfaces/automated-testing/scd/scd.yaml

```diff
@@ -35,7 +35,478 @@ components:
+      default: 0
+    ################################################################################
+    ####################### End of ASTM-standard definitions #######################
+    ################################################################################
+
+    OperationalIntentTestInjection:
+      description: >-
+        Parameters that define an operational intent: this injection is used to
+        create a operational intent reference in the DSS and also responding to 
+        requests for details of that operational intent (by other USSes or the 
+        test driver). The USS under test will need to process this data to both
+        create a valid operational intent reference and responding to a query for  
+        details. 
+      required:
+      - state
+      - priority
+      - volumes
+      - off_nominal_volumes
+      
+      type: object
+      properties:
+        state:
+          $ref: '#/components/schemas/OperationalIntentState'
+        priority:
+          $ref: '#/components/schemas/Priority'
+        volumes:
+          description: >-
+            Nominal volumes, as would be reported by a USS's operational_intents endpoint.
+          type: array
+          items:
+            $ref: '#/components/schemas/Volume4D'
+        off_nominal_volumes:
+          description: >-
+            Off-Nominal volumes, as would be reported by a USS's operational_intents endpoint.
+          type: array
+          items:
+            $ref: '#/components/schemas/Volume4D'
+
+    FlightAuthorisationData:
+      description: |-
+        A dataset to hold details of a UAS flight authorization request matching some of the requirements of Annex IV of the U-Space regulation 2021/664
+      type: object
+      properties:
+        uas_serial_number:
+          type: string
+          example: "T4S6BBMSZV4H0650"
+          description: Serial number of the UAS per the CTA-2063-A standard
+        operation_mode:   
+          $ref: '#/components/schemas/OperationMode'
+        operation_category:
+          type: string
+          example: "normal"
+          description: |-
+            Specify if the flight is "special" or has a specific priority.
+        uas_class: 
+          $ref: '#/components/schemas/UASClass'
+        identification_technologies: 
+          type: array
+          items:
+            type: string
+            enum:
+              - Undeclared
+              - Network
+              - Broadcast
+            description: Specify an identification technology that will be used for this flight
+        connectivity_methods:
+          type: array
+          items:
+            type: string
+          example: ['radio']
+        endurance_minutes: 
+          type: number
+          description: A number to specify the endurance of the drone in minutes
+        emergency_procedure_url:
+          type: string
+          format: url
+          description: A valid url to contact in case of emergencies
```

Comment:
Editoral suggestion:
```
            The URL at which the applicable emergency procedure in case of a loss of command and
            control link may be retrieved.
            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 9.
```

Reply:
Thanks! I have now modified this.